tests/unittests/Foundation/NSPointerArrayTests.mm

```diff
@@ -259,8 +243,14 @@ @implementation NSPointerArraySubClass
     ASSERT_EQ_MSG(array.count, 2, @"Incorrect size");
 }
 
-TEST(NSPointerArray, Weak) {
-    NSPointerArray* array = [NSPointerArray weakObjectsPointerArray];
+// TODO::
+// Bug #803:NSArrayPointer should implement dealloc observer for objects added to it
+// Disabling test until the bug is fixed.
+//
+DISABLED_TEST(NSPointerArray, Weak) {
```

Comment:
Fully disable as this test now fails even on our platform. The test was written wrong

---
In reply to: [73436821](https://github.com/Microsoft/WinObjC/pull/806#discussion_r73436821) [](ancestors = 73436821)